test/index.js

```diff
@@ -243,7 +248,12 @@ describe('hapi-setup Plugin', function () {
           },
           attributes: {
             name: 'bar',
-            version: '1.0.0'
+            version: '1.0.0',
+            pkg: {
+              version: '0.0.0'
+            },
+            multiple: false,
```

Comment:
Did you need to add these? I'm pretty sure hapi defaults them if they are missing.